### ISSUE TITLE
Parameterized query doesn't work

### ISSUE THREAD

**Issue by nallwhy** [livebook-dev/req_athena#30]
```elixir
credential = [...]

Req.new()
|> ReqAthena.attach(credential)
|> Req.post!(athena: {"SELECT ?", [1]})
```

throws error

```
** (CaseClauseError) no case clause matching: {nil, false}
    (req_athena 0.1.2) lib/req_athena.ex:188: ReqAthena.handle_athena_result/1
    (req 0.3.5) lib/req/request.ex:705: anonymous fn/2 in Req.Request.run_response/2
    (elixir 1.14.2) lib/enum.ex:4751: Enumerable.List.reduce/3
    (elixir 1.14.2) lib/enum.ex:2514: Enum.reduce_while/3
    (req 0.3.5) lib/req/request.ex:704: Req.Request.run_response/2
    (req 0.3.5) lib/req.ex:327: Req.post!/2
    (req_athena 0.1.2) lib/req_athena.ex:271: ReqAthena.execute_prepared_query/1
    #cell:5eknz4ezrwqs47sqb5n4owthwp2eqjr3:6: (file)
```

**Comment by nallwhy** [comment]
@aleDsz This happens with req >= 0.3.2

**Comment by nallwhy** [comment]
`IO.inspect({action, parameterized?})` in `handle_athena_result/2`

```
{"StartQueryExecution", true}
{"GetQueryExecution", true}
{"GetQueryResults", true}
{nil, false}
```

**Comment by nallwhy** [comment]
duplicated response step is not appended

**Comment by aleDsz** [comment]
Fixed with #31 PR. I'll ship a new version 💜
